tests/peft/test_peft_onnx_transforms.py

```diff
@@ -0,0 +1,63 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+import textwrap
+
+import onnx
+
+from QEfficient.peft.onnx_transforms import AdapterWeightsToInputsTransform
+
+
+def test_adapter_weights_to_inputs_transform():
+    external_tensors_file = "weight.raw"
+    adapter_name = "testAdapter1"
+    test_onnx = onnx.parser.parse_model(f"""
+    <
+        ir_version: 8,
+        opset_import: ["" : 17]
+    >
+    test_adapter_weights (float[n, 32] input) => (float[n, 32] output)
+    <
+        float[32, 32] layer1_{adapter_name}_weight = [ "location" : "{external_tensors_file}" ],
+        float[32, 32] layer2_{adapter_name}_weight = [ "location" : "{external_tensors_file}" ]
+    >
+    {{
+        layer1output = MatMul (input, layer1_{adapter_name}_weight)
+        output = MatMul (layer1output, layer2_{adapter_name}_weight)
+    }}
+    """)
+
+    out_onnx, transformed = AdapterWeightsToInputsTransform.apply(test_onnx, adapter_name=adapter_name)
+    assert not transformed
```

Comment:
did you want to remove these lines

Reply:
No, this also is a part of the transform test that it will not touch a model that doesn't match the exact criteria.